lib/adhearsion/script_ahn_loader.rb

```diff
@@ -8,9 +8,22 @@ module ScriptAhnLoader
     RUBY = File.join(*RbConfig::CONFIG.values_at("bindir", "ruby_install_name")) + RbConfig::CONFIG["EXEEXT"]
     SCRIPT_AHN = File.join('script', 'ahn')
 
+    def self.load_script_ahn(path = Dir.pwd)
+      path = Pathname.new(path).expand_path
+      begin
+        script = File.join(path, SCRIPT_AHN)
```

Comment:
OK will do as recommended if it reads better ... and squash all relevant commits with a change-log